Jenkinsfile

```diff
@@ -4,4 +4,5 @@ common {
   upstreamProjects = 'confluentinc/common'
   pintMerge = true
   downStreamValidate = false
+  nodeLabel = 'docker-debian-jdk8'
```

Comment:
This change is to prevent builds from failing.

Reply:
Any reference on why we needed to add this now?
Anything changed recently?

Reply:
@subhashiyer9 [This](https://github.com/confluentinc/kafka-overlay/pull/176#issue-1824790544) description somewhat explains the change